src/api/public/apidocs-new/paths/source_project_name_meta.yaml

```diff
@@ -33,3 +33,50 @@ get:
       $ref: '../components/responses/unknown_project.yaml'
   tags:
     - Sources - Projects
+
+put:
+  summary: Write project meta file
+  description: |
+    Write project's meta file. Create the project if it doesn't exist.
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+    - in: query
+      name: comment
+      required: false
+      schema:
+        type: string
+      description: Comment that explains the changes you made in meta file.
+  requestBody:
+    description: Project meta definition.
+    content:
+      application/xml; charset=utf-8:
+        schema:
+          $ref: '../components/schemas/source/project_meta.yaml'
+        example:
+          name: home:Admin
+          title: Endless Night
+          description: Test project
+          person:
+            userid: Admin
+            role: maintainer
+          repository:
+            name: openSUSE_Tumbleweed
+            path:
+              project: openSUSE.org:openSUSE:Factory
+              repository: snapshot
+            arch:
+              - x86_64
+              - aarch64
+  responses:
```

Comment:
I miss a few 403 responses here. I don't think we need to list all of them but at least one...

when the project is locked (see [this line](https://github.com/openSUSE/open-build-service/blob/7e5cf6630d2e5202ca5dd4796c66010560b8b234/src/api/app/controllers/source_project_meta_controller.rb#L46)), the response is
```
<status code="change_project_no_permission">
  <summary>The project home:Admin is locked</summary>
</status>
```

when the user doesn't have permission to modify the project (see [this line](https://github.com/openSUSE/open-build-service/blob/7e5cf6630d2e5202ca5dd4796c66010560b8b234/src/api/app/controllers/source_project_meta_controller.rb#L49)), the response is:
```
<status code="change_project_no_permission">
  <summary>no permission to change project</summary>
</status>
```

when the user is not allowed to create a new project (see [this line](https://github.com/openSUSE/open-build-service/blob/7e5cf6630d2e5202ca5dd4796c66010560b8b234/src/api/app/controllers/source_project_meta_controller.rb#L55)), the response is:
```
<status code="create_project_no_permission">
  <summary>no permission to create project home:yyy</summary>
</status>
```

There are more responses like these ones as a result of calling the methods `ensure_access_to_edit_remote_project`, `ensure_xml_attributes_are_valid`, `remove_repositories!`
starting in [line 60](https://github.com/openSUSE/open-build-service/blob/7e5cf6630d2e5202ca5dd4796c66010560b8b234/src/api/app/controllers/source_project_meta_controller.rb#L60).

Reply:
Done. I've documented this response